bigbluebutton-html5/tests/webdriverio/specs/login.spec.js

```diff
@@ -4,18 +4,27 @@ let LandingPage = require('../pageobjects/landing.page');
 let chai = require('chai');
 
 describe('Landing page', function () {
+
+  beforeEach(function () {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000; // default value is 10000
+  });
+
   it('should have correct title', function () {
     LandingPage.open();
-    chai.expect(browser.getTitle()).to.equal(LandingPage.title);
+
+    chai.expect(chromeBrowser.getTitle()).to.equal(LandingPage.title);
+    chai.expect(firefoxBrowser.getTitle()).to.equal(LandingPage.title);
```

Comment:
Cant you write a function that test the same thing against both browsers? You are just changing a single variable every test case to test with both browsers, seems overly repetitive to me.